lib/amazonka/amazonka.cabal

```diff
@@ -112,7 +112,15 @@ library
     Network.AWS.Data.XML
     Network.AWS.Endpoint
     Network.AWS.Error
+    Network.AWS.HTTPUnsigned
     Network.AWS.Pager
+    Network.AWS.STS
+    Network.AWS.STS.AssumeRoleWithWebIdentity
+    Network.AWS.STS.Lens
+    Network.AWS.STS.Types
+    Network.AWS.STS.Types.AssumedRoleUser
+    Network.AWS.STS.Types.PolicyDescriptorType
+    Network.AWS.STS.Waiters
```

Comment:
Perhaps these should go into `other-modules`, so we don't force people using `amazonka-sts` to package-qualify their imports?

Reply:
They are already in `other-modules`, I didn't add any new exposed modules.

Reply:
OK, thanks for checking.